frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -319,5 +321,5 @@ action.GitMachete.ToggleListingCommitsAction.description.disabled.no-branches=To
 action.GitMachete.OpenMacheteTabAction.text=Open Git Machete Tab
 action.GitMachete.OpenMacheteTabAction.description=Open Git Machete Tab
 
-action.GitMachete.OpenMacheteTabAction.notification.fail.title=Could not open Git Machete tab
-action.GitMachete.OpenMacheteTabAction.notification.fail.message.no-git=Ensure that Git VCS is enabled in the project
+action.GitMachete.OpenMacheteTabAction.notification.title.could-not-open-tab=Could not open Git Machete tab
```

Comment:
ok, so what's about this rule:
> We use `cannot` if "it" (whatever IT is) can be never performed but `could not` if "it" cannot due to some exceptional circumstances.

Hence:
- `Cannot create file 'machete': Directory with the same name exists` b/c it is never possible while the directory exists
- `Could not open Git Machete tab` b/c we are not able to specify why it was not opened; it could be missing tool window or missing content(tab) but we neither want to expose the technical terms nor we know the actual reason why the tool window or the content tab is null; most likely it's disabled Git Vcs

Reply:
Ok! Pls apply & document in .properties file or in CONTRIBUTING.md